Modules/_io/bytesio.c

```diff
@@ -30,17 +30,34 @@ typedef struct {
   * exports > 0.  Py_REFCNT(buf) == 1, any modifications are forbidden.
 */
 
+static int
```

Comment:
Should these be marked as inlined?

Reply:
Compilers can inline the function even if it is not marked as `inline`.
No need to mark `inline` explicitly unless the very critical part.